tool/tctl/common/collection.go

```diff
@@ -723,6 +781,54 @@ func (c *windowsDesktopCollection) writeText(w io.Writer) error {
 	return trace.Wrap(err)
 }
 
+type windowsDesktopAndService struct {
+	desktop types.WindowsDesktop
+	service types.WindowsDesktopService
+}
+
+type windowsDesktopAndServiceCollection struct {
+	desktops []windowsDesktopAndService
+	verbose  bool
+}
+
+func (c *windowsDesktopAndServiceCollection) writeText(w io.Writer) error {
+	var rows [][]string
+	for _, d := range c.desktops {
+		labels := d.desktop.GetAllLabels()
+		if !c.verbose { // remove teleport.dev labels unless we're in verbose mode.
+			for key := range labels {
+				if strings.HasPrefix(key, types.TeleportNamespace) {
+					delete(labels, key)
+				}
+			}
+		}
```

Comment:
I would make this into a helper method and used for displaying other resources as well since they also can have teleport.dev labels, so it's consistent.